include/ensmallen_bits/sa/sa_impl.hpp

```diff
@@ -219,12 +220,12 @@ inline void SA<CoolingScheduleType>::MoveControl(const size_t nMoves,
                                                  MatType& accept,
                                                  MatType& moveSize)
 {
-  MatType target;
-  target.copy_size(accept);
+  MatType target(accept.n_rows, accept.n_cols);
```

Comment:
```suggestion
  MatType target(accept.n_rows, accept.n_cols, GetFillType<MatType>::none);
```
A small optimization.